package.json

```diff
@@ -70,8 +71,8 @@
     "jest": "^29.7.0",
     "jest-plugin-context": "^2.9.0",
     "prettier": "^2.3.2",
-    "serverless": "^3.38.0",
-    "serverless-offline": "^12.0.4",
+    "serverless": "^3.0.0",
```

Comment:
Did this need to be downgraded, was there unexpected behaviour / an error?

Reply:
Yes, it is - I'm just trying to understand the rationale for making this change at all?

Reply:
With v4 you need a licence/key, so yeah would expect us to say at 3.x!

Reply:
Good to know haha!